Calling/ClientApp/src/core/sideEffects.ts

```diff
@@ -108,6 +121,11 @@ const subscribeToParticipant = (
   
         if (addedStream.isAvailable) { dispatch(addScreenShareStream(addedStream, participant)); }
       }
+      else if (addedStream.mediaStreamType === 'Video') {
+        addedStream.on('isAvailableChanged', () => {
+          remoteStreamSelector.participantVideoChanged(utils.getId(participant.identifier), addedStream.isAvailable);
```

Comment:
is there a chance the state has already changed to available before we're ready? can we also query the state of the model right away as well?

Reply:
It will be changed to available before we process and update the redux selected list, but UI will not render it since it needs the `isParticipantStreamSelected` boolean to be true before start rendering. Does this answer your question?

Reply:
Ok